src/cocaine-app/__init__.py

```diff
@@ -274,6 +275,11 @@ def init_job_finder():
     return jf
 
 
+def init_group_history_finder():
+    ghf = history.GroupHistoryFinder(meta_db)
+    return ghf
```

Comment:
This is common for init_* functions in __init__.py. Temporary object may be used to add some cocaine handlers, so I define such objects uniformly.